internal/cueutils/convert/to_testapi/from_sarif.cue

```diff
@@ -140,7 +140,14 @@ _findings: list.Sort(list.Concat([ for run in input.runs {
                     }
                 }
             }]
+            if result.suppressions != _|_ {
+            	if result.suppressions[0].justification != _|_ {
```

Comment:
Do we care if there's no justification given? I think we'd still want to honor the ignore, if the finding is suppressed.

```suggestion
            if result.suppressions != _|_ {
              if len(result.suppressions) > 0 {
```